knimin/lib/tests/test_geocoder.py

```diff
@@ -72,7 +72,7 @@ def test_geocode_nonmock(self):
         obs = geocode('9500 Gilman Dr, La Jolla, CA')
         exp = Location('9500 Gilman Dr, La Jolla, CA', 32.8794239,
                        -117.2369135, 114.7895736694336, 'San Diego', 'CA',
-                       '92037', 'United States')
```

Comment:
I'm unsure if it changed or there was something else going on in the code before. I just look up the address and the zipcode is 92093